Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/mpi/adapter/component/hl7parsers/HL7DbParser201305.java

```diff
@@ -202,33 +191,33 @@ public static Timestamp extractBirthdate(PRPAMT201306UV02ParameterList params) {
                             } else {
                                 nameString = strValue;
                             }
-                            LOG.info("nameString=" + nameString);
+                            LOG.info("nameString{}", nameString);
```

Comment:
when you used parameter in the log add a separator for readability "nameString: {}"